### PR TITLE
ENG-6207 - RN Force Start

### PR DESCRIPTION
**Pull request by kevinsites**

### CODE REVIEW

File: android/src/main/java/com/neuroidreactnativesdk/NeuroidReactnativeSdkModule.kt [hunk 1]
```diff
@@ -38,6 +38,11 @@ class NeuroidReactnativeSdkModule(reactContext: ReactApplicationContext) :
             NeuroID.setNeuroIdInstance(neuroID)
         }
         NeuroID.getInstance()?.configureWithOptions(key, null)
+
+        val currentActivity = currentActivity
```
Comment (kishisaka-nid) [inline android/src/main/java/com/neuroidreactnativesdk/NeuroidReactnativeSdkModule.kt:42]: not sure you need this.

File: android/build.gradle [hunk 2]
```diff
@@ -131,6 +131,7 @@ dependencies {
   implementation 'androidx.core:core-ktx:1.6.0'
   releaseImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk:master-SNAPSHOT'
   debugImplementation 'com.github.neuro-id.neuroid-android-sdk:react-android-sdk-debug:master-SNAPSHOT'
+  implementation 'com.google.code.gson:gson:2.10.1'
```
Comment (kishisaka-nid) [inline android/build.gradle:134]: I had to do this as well. Looks ok to me!

**Review by kishisaka-nid — APPROVED** [review]
remove self assignment, I am sure it is not needed. otherwise, 🚢

**Comment by kishisaka-nid** [comment]
LGTM. 🚢

**Review by priya-xavier — APPROVED** [review]

**Pull request merged**